src/axolotl/utils/models.py

```diff
@@ -627,7 +628,7 @@ def load_model(
         for name, module in model.named_modules():
```

Comment:
Does the above if statement need to check for phi/gpt neox? How about miXtral?

Reply:
it shouldn't since it's reusing `embedding_modules` defined in the same function from line 591.

Reply:
mistral, mixtral, and llama all share the same embedding layers, which is the default return value for `get_linear_embedding_layers(...)`

Reply:
Ah sorry, I meant this.

```
    if needs_fa2_dtype or (
        cfg.flash_attention
        and (cfg.is_llama_derived_model or cfg.is_mistral_derived_model)
    ):
    ```
    
    GH didn't show this part.

Reply:
ah, good catch, I've updated that now. thanks!